src/panels/lovelace/components/hui-card-options.ts

```diff
@@ -99,6 +99,10 @@ export class HuiCardOptions extends LitElement {
           rgba(0, 0, 0, 0.12) 0px 1px 5px -4px,
           rgba(0, 0, 0, 0.2) 0px 3px 1px -2px;
         display: flex;
+        position: relative;
+        margin-top: -1px;
+        border-radius: 0px 0px var(--ha-card-border-radius, 2px)
```

Comment:
This works but now you're including defaults from somewhere else, that is bound to go out of sync when the defaults are updated . 

Instead just set `border-top-left-radius: 0` and `border-top-right-radius: 0`